charactersheet/charactersheet/templates/common/wizard_profile_step.tmpl.html

```diff
@@ -112,6 +112,20 @@
             </div>
           </div>
         </div>
+        <div class="h3">Adventuring Gear.<br />
+          <small>Select your backpack and all of your items will pre-populate in the inventory module.</small>
+        </div>
+        <div class="row">
+          <div class="col-md-6 col-padded">
+            <div class="input-group">
+              <span class="input-group-addon" id="basic-addon2">Backpack</span>
+              <input type="text" class="form-control" aria-describedby="basic-addon2"
```

Comment:
For races, weapons, and traits a text input makes sense, but there are a finite amount of backpacks we will auto-populate. This field does not do anything if they type in `banana`, so I propose this input become a select with all of the backpacks we support.